flare_emu.py

```diff
@@ -37,10 +38,7 @@
 MAX_ALLOC_SIZE = 10 * 1024 * 1024
 MAXCODEPATHS = 20
 MAXNODESEARCH = 100000
-try:
-    long        # Python 2
-except NameError:
-    long = int  # Python 3
+long = int() if platform.python_version_tuple()[0] is 3

```

Comment:
This change, while being more specific, makes the code less useful because it only sets the 'long' type if under Python3. Currently, the code will set the 'long' type under any condition in which it is not already set, making it more robust.

Also, you have a bug in the code: it should be `long = int` and not `long = int()`. Your code is setting the variable 'long' to the int value `0`